### ISSUE TITLE
highlight band dimension should go with font size

### ISSUE THREAD

**Issue by edoardoc** [adad184/MMNavigatorFont#3]
as u can see from attached screenshot, increasing font size doesn't increase the blue selector band accordingly and bigger letters gets cut in the bottom.

![screen shot 2016-03-24 at 23 02 10](https://cloud.githubusercontent.com/assets/368717/14031254/a2285c22-f214-11e5-819c-15473800a2dd.png)

**Comment by adad184** [comment]
yep, it's a known problem. :)   

no idea how to hook & modify it.   Any idea?

**Comment by edoardoc** [comment]
if I am not mistaken everything happens thanks to 

`[objc_getClass("DVTTableCellViewOneLine") aspect_hookSelector:@selector(awakeFromNib)
                                                      withOptions:AspectPositionAfter
                                                       usingBlock:fontBlock
                                                            error:nil];
`

if so I would try and play with AspectPositionAfter

**Comment by edoardoc** [comment]
but I have no idea if this would be the right way to go :)

**Comment by adad184** [comment]
the `rowHeight` of `NSOutlineView` can change the height of cell
but it only effect when outlineView initialized.
it's not work when i change the font.

and though i can change the height, still lot of problem should work out.
the frame of the titleLabel and detailTitleLabel of the cell must be adjusted.

so i suggest not to choose a big size font. :)

**Comment by edoardoc** [comment]
are u talking about property

@property (nonatomic, weak)   NSView *outlineView;

in XcodeCustomizeFont.m ?

**Comment by adad184** [comment]
yes, it's a type of `NSOutlineView`, which is a subclass of `NSTableView`

**Comment by edoardoc** [comment]
check out this:
http://stackoverflow.com/questions/7504546/view-based-nstableview-with-rows-that-have-dynamic-heights?answertab=active#tab-top
it looks there is a way to do it, without setting the height in the init section of NSOutlineView.
plus, if you set a bigger font, you will see that some of the tabs of the navigator already do that: you can see it clearly in the issue tab (see
![screen shot 2016-03-27 at 23 58 38](https://cloud.githubusercontent.com/assets/368717/14067858/f134040a-f477-11e5-84b6-f799bce26eb2.png)
 screenshot)
